.github/workflows/build-and-publish.yml

```diff
@@ -155,7 +155,7 @@ jobs:
       - name: Bundle universal JAR
         run: ./gradlew lib:jar
       - name: Rename JAR
-        run: mv lib/build/libs/lib-[0-9].[0-9][0-9].[0-9].jar starknet.jar
+        run: mv lib/build/libs/lib-*.jar starknet.jar
```

Comment:
note: After running `./gradlew lib:jar` only one file (with lib jar) will be present so it's safe to widen the pattern.